odl/solvers/advanced/proximal_operators.py

```diff
@@ -1079,6 +1080,91 @@ def _call(self, x, out):
     return ProximalCConjKL
 
 
+def proximal_cconj_kl_version_two(space, lam=1, g=None):
+    """Proximal operator factory of the convex conjugate of the KL divergence.
```

Comment:
Should also be adapted somewhat.
```
"""Factory for the convex conjugate proximal of the "second kind" KL divergence."""
```
perhaps? Could be too long though. Alternatively "Convex conjugate proximal factory of ..." shortens a bit.